test/Microsoft.ML.Tests/PermutationFeatureImportanceTests.cs

```diff
@@ -37,18 +38,31 @@ public void TestPfiRegressionOnDenseFeatures(bool saveModel)
             var model = ML.Regression.Trainers.OnlineGradientDescent().Fit(data);
 
             ImmutableArray<RegressionMetricsStatistics> pfi;
-            if(saveModel)
+            ImmutableDictionary<string, RegressionMetricsStatistics> pfiDict;
+
+            if (saveModel)
             {
                 var modelAndSchemaPath = GetOutputPath("TestPfiRegressionOnDenseFeatures.zip");
                 ML.Model.Save(model, data.Schema, modelAndSchemaPath);
 
                 var loadedModel = ML.Model.Load(modelAndSchemaPath, out var schema);
                 var castedModel = loadedModel as RegressionPredictionTransformer<LinearRegressionModelParameters>;
+
+                // PFI changes the random state, so we need to reset it and create another seed for both PFI to match
+                ML = new MLContext(42);
+                var ml2 = new MLContext(42);
+
                 pfi = ML.Regression.PermutationFeatureImportance(castedModel, data);
+                pfiDict = ml2.Regression.PermutationFeatureImportance(loadedModel, data);
             }
             else
             {
+                // PFI changes the random state, so we need to reset it and create another seed for both PFI to match
+                ML = new MLContext(42);
+                var ml2 = new MLContext(42);
+
                 pfi = ML.Regression.PermutationFeatureImportance(model, data);
+                pfiDict = ml2.Regression.PermutationFeatureImportance((ITransformer)model, data);
```

Comment:
Yes. The issue is that the PFI method (of the same name) takes a `ISingleFeaturePredictionTransformer<TModel>`, and so the compiler defaults to that method instead of the method that just takes an ITransformer. `model` in this case is just a single estimator so its the actual type instead of ITransformer. Is there another way to work around that?